pom.xml

```diff
@@ -88,6 +88,7 @@
         <version.jboss-integration-platform>6.0.10.Final</version.jboss-integration-platform>
 
         <!-- Others -->
+        <commons.collections4.version>4.1</commons.collections4.version>
```

Comment:
Apache commons collections 4 are not available in wildfly, this would need a new dependency which is not a trivial task. I think the only reason for including it is using Radix (Patricia) tree.